airflow/config_templates/default_airflow.cfg

```diff
@@ -703,6 +703,11 @@ gcp_service_account_keys =
 # It will raise an exception if called from a process not running in a kubernetes environment.
 in_cluster = True
 
+# Use the cluster_context or config_file to run outside of cluster
+cluster_context =
+config_file =
```

Comment:
Are these actually needed? Is setting `in_cluster = False` enough?

Reply:
Yeah these are required to initialise kube_client if in_cluster is set to False

Reply:
They aren't though - they are optional. If they aren't provided then the Python Kube library will look at the default config file and use the currently selected context. We should mirror that behaviour.